pkg/controller/replication/replication_controller.go

```diff
@@ -66,6 +67,10 @@ type ReplicationManager struct {
 	kubeClient clientset.Interface
 	podControl controller.PodControlInterface
 
+	// personalPodInformer is used to hold a personal informer.  If we're using a normal shared informer, then the informer will be
+	// started for us.  If we have a personal informer, we must start it ourselves
+	personalPodInformer framework.SharedInformer
```

Comment:
I'm wondering if "personalPodInformer" is the best name. Maybe "internalPodInformer"?

Also, the same comment about comment as in endpoint controller.